hedera-mirror-web3/src/test/java/com/hedera/mirror/web3/repository/EntityRepositoryTest.java

```diff
@@ -0,0 +1,57 @@
+package com.hedera.mirror.web3.repository;
+
+import static com.hedera.mirror.common.domain.entity.EntityType.ACCOUNT;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.google.common.collect.Range;
+
+import com.hedera.mirror.common.domain.entity.Entity;
+import com.hedera.mirror.common.util.DomainUtils;
+import com.hedera.mirror.web3.Web3IntegrationTest;
+
+import java.time.Instant;
+import javax.annotation.Resource;
+import org.junit.jupiter.api.Test;
+
+class EntityRepositoryTest extends Web3IntegrationTest {
+
+    private final Instant now = Instant.now();
+
+    private Long entityId = 78L;
+
+    private static final byte[] KEY = new byte[33];
+
+    @Resource
+    private EntityRepository entityRepository;
+
+    @Test
+    void findByIdAndDeletedIsFalseSuccessfulCall() {
+        Entity entity = entity();
+        entityRepository.save(entity);
+        final var result = entityRepository.findByIdAndDeletedIsFalse(entityId);
+        assertThat(result).get().isEqualTo(entity);
+    }
+
+    @Test
+    void findByAliasAndDeletedIsFalseSuccessfulCall() {
+        Entity entity = entity();
+        entityRepository.save(entity);
+        final var result = entityRepository.findByAliasAndDeletedIsFalse(KEY);
```

Comment:
Mirror node standard is to use `DomainBuilder` everywhere to generate default objects and customize it only as necessary for the scenario. This avoids hardcoding fields and boilerplate setup.

```suggestion
        var entity = domainBuilder.entity().persist();
        final var result = entityRepository.findByAliasAndDeletedIsFalse(entity.getAlias());
```